### DISCUSSION TITLE
Welcome to YASA Discussions!

### DISCUSSION THREAD

**Discussion by raphaelvallat** [raphaelvallat/yasa#52] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of the YASA community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by umair-hassan** [comment]
Hi Raphael, thanks for building YASA, regarding the ndPAC changes in the latest release, would this also comprise negative values still like previous releases? Can you write a few lines for me to better understand how the negative values can be interpreted from this release then, many thanks, Umair

**Reply by umair-hassan** [reply]
Sorry, I confused this with sigma phase property, all good!

**Comment by umair-hassan** [comment]
BTW just another question, the sp_detect also have those spindles that were just happening along with Slow waves as well, right? 

I am interested in finding out Spindles that happen with slow wave and those Spindles that do not couple with the Slow wave, can you may be give me a hint, how to do it with lesser manual work?

**Reply by raphaelvallat** [reply]
Hi @umair-hassan,

This is possible with the https://raphaelvallat.com/yasa/build/html/generated/yasa.SWResults.html#yasa.SWResults.find_cooccurring_spindles. You first need to run the slow-waves detection, then spindles detection and then apply the find_cooccurring_spindles function.

Hope this helps!
Raphael
